### ISSUE TITLE
error: unresolved import `std::os::unix`. There is no `unix` in `std::os` [E0432]

### ISSUE THREAD

**Issue by Ga1der** [blackbeam/rust-mysql-simple#70]
C:\Users\_\.cargo\bin\cargo.exe run --bin rust
   Compiling openssl v0.7.14
   Compiling openssl-sys-extras v0.7.14
   Compiling mysql v7.1.2
C:\Users\_\.cargo\registry\src\github.com-1ecc6299db9ec823\mysql-7.1.2\src\io.rs:24:5: 24:26 error: unresolved import `std::os::unix`. There is no `unix` in `std::os` [E0432]
C:\Users\_\.cargo\registry\src\github.com-1ecc6299db9ec823\mysql-7.1.2\src\io.rs:24 use std::os::unix as unix;
                                                                                        ^~~~~~~~~~~~~~~~~~~~~
C:\Users\_\.cargo\registry\src\github.com-1ecc6299db9ec823\mysql-7.1.2\src\io.rs:24:5: 24:26 help: run `rustc --explain E0432` to see a detailed explanation
C:\Users\_\.cargo\registry\src\github.com-1ecc6299db9ec823\mysql-7.1.2\src\conn/mod.rs:64:5: 64:18 error: unresolved import `std::os::unix`. There is no `unix` in `std::os` [E0432]
C:\Users\_\.cargo\registry\src\github.com-1ecc6299db9ec823\mysql-7.1.2\src\conn/mod.rs:64 use std::os::unix;
                                                                                              ^~~~~~~~~~~~~
C:\Users\_\.cargo\registry\src\github.com-1ecc6299db9ec823\mysql-7.1.2\src\conn/mod.rs:64:5: 64:18 help: run `rustc --explain E0432` to see a detailed explanation
error: aborting due to 2 previous errors
error: Could not compile `mysql`.

To learn more, run the command again with --verbose.

Process finished with exit code 101

**Comment by blackbeam** [comment]
You should look into "[Windows support](https://github.com/blackbeam/rust-mysql-simple#windows-support-since-0180)" section of a README.md.

I.e. you need to disable `ssl` feature on Windows:

```toml
[dependencies.mysql]
version = <your version here>
default-features = false
features = ["pipe"]
```

**Comment by DenisKolodin** [comment]
I've fixed it with PR #71
